src/OSPSuite.Core/Comparison/CalculationMethodDiffBuilder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace OSPSuite.Core.Comparison
+{
+   public class CalculationMethodDiffBuilder : DiffBuilder<CalculationMethod>
+   {
+      public override void Compare(IComparison<CalculationMethod> comparison)
+      {
+         //No need to compare name and display name as they will be correlated
+         CompareStringValues(x => x.DisplayName, x => x.DisplayName, comparison);
```

Comment:
I have ensured that DisplayName is never empty so we can really compare only display name in this case. Category is going to give the context